atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/ranges/DbKvsGetRanges.java

```diff
@@ -191,16 +191,16 @@ public DbKvsGetRanges(DbKvs kvs,
             String query = String.format(
                     SIMPLE_ROW_SELECT_TEMPLATE,
                     tableName,
```

Comment:
I find it a bit odd that we report in the comment a different (read: non-prefixed) table name and then operate on the prefixed table name.  Seems like it would be more clear / helpful if the comment included the prefix name.